src/Altinn.App.Core/Features/Validation/Wrappers/TaskValidatorWrapper.cs

```diff
@@ -42,8 +39,8 @@ public Task<List<ValidationIssue>> Validate(IInstanceDataAccessor dataAccessor,
     /// <inheritdoc />
     public Task<bool> HasRelevantChanges(IInstanceDataAccessor dataAccessor, string taskId, DataElementChanges changes)
     {
-        throw new NotImplementedException(
-            "TaskValidatorWrapper should not be used for incremental validation, because it sets NoIncrementalValidation to true"
-        );
+        // If task validator sets <see cref="ITaskValidator.NoIncrementalValidation"/> to false, it will run on every PATCH request, and because <see cref="ITaskValidator" /> does not have a HasRelevantChanges method,
```

Comment:
Denne setningen var veldig vanskelig å lese. Kan dette potensielt være litt tydeligere, hvis det er riktig? 
```suggestion
        /// We set HasRelevantChanges to true to align with <see cref="ITaskValidator.NoIncrementalValidation"/>.
        /// Since ITaskValidator does not define HasRelevantChanges, we must choose a value:
        /// - false would prevent validation on PATCH requests.
        /// - true ensures validation runs on PATCH when NoIncrementalValidation is false.
```

Reply:
Det føles ikke helt riktig det heller.

Jeg returnerer `true` fordi det ikke er noen måte å vite om det er relevante endringer (metoden mangler i interface). Vanligvis kjører ikke `ITaskValidator` på PATCH, men siden denne PR en gjør det mulig å sette `NoIncrmentalValidation => false` så kan det hende noen gjør det, og da må vi anta den alltid har relevante endringer.